docs/en/command/cacct.md

```
@@ -1,276 +1,403 @@
+**--type=&lt;type1,type2,...&gt;**
+
+:   **Applies to:** `job`, `step`  
+Specify task types to view (comma-separated list). Valid values: 'Interactive', 'Batch', 'Container'. Default: all
+types.
+
+**-F, --full**
+
+:   **Applies to:** `job`, `step`  
+Display full information without field truncation. By default, only 30 characters per cell are displayed.
+
+**-N, --noheader**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)